### PR TITLE
Bug 1832293, update metrics for Pods per node

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1832293

### CODE REVIEW

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Comment by chaitanyaenr** [comment]
@RobertKrawitz PTAL.

**Comment by RobertKrawitz** [comment]
/hold

I'm in the process of testing 500 pods/node on 4.4, but we don't want to increase 4.3 without also increasing 4.4.

**Comment by RobertKrawitz** [comment]
500 pods/node test succeeded on 4.4.3.  So,
/hold cancel

File: modules/openshift-cluster-maximums.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@
 | Number of Pods per node
 | 250
 | 250
-| 250
+| 500
 | 250
```
Comment (RobertKrawitz) [inline modules/openshift-cluster-maximums.adoc:28]: Needs to go to 500.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

**Comment by RobertKrawitz** [comment]
I see the 4.3 tested max was raised; the 4.4 does also.

**Comment by ahardin-rh** [comment]
@RobertKrawitz Updated. Thanks!

**Pull request merged**